server/api/PosterHandler.cs

```diff
@@ -23,9 +23,11 @@ public override void Process(ResourceManager resourceManager)
                 Style style = Style.Poster;
                 ParseOptions(ref options, ref style);
                 string title;
+                bool compositing = GetBoolOption("compositing", false);
                 bool clipOutsectorBorders;
-                bool transparent = false;
+                bool transparent = compositing ? true : false; // compositing requires the returned poster is a png for transparency
```

Comment:
Ideally compositing should use svg to avoid artifacts when scaling custom sector posters to match tile scale, however I'm not sure how to make the required cutouts along the edge of the poster when generating an svg.